packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/migrations/transform_set_managed_default.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SavedObjectUnsanitizedDoc } from '@kbn/core-saved-objects-server';
+
+export const transformSetManagedDefault = (
+  doc: SavedObjectUnsanitizedDoc<unknown> | undefined
+) => ({
+  transformedDoc: { ...doc!, managed: doc!.managed ?? false },
```

Comment:
Good question! As it stands, the UI will need to implement the logic to handle different functionality based on if `managed` is true or not (or undefined). However, we only really need the flag for `fleet` right now and don't intend to allow setting `managed` for any other plugin, otherwise we'd need to build in a `managed_by` option too. Is it worth the extra effort? I don't know. @rudolf wdyt?

Reply:
We're also already migrating all documents in 8.8.0 so this would not introduce additional downtime. But if there was a downside to doing it I agree it could have been left out.

Reply:
@jloleysens  are you ok with keeping the migration for now?